pkg/expr/mathexp/type_series.go

```diff
@@ -48,8 +48,29 @@ func SeriesFromFrame(frame *data.Frame) (s Series, err error) {
 			s.ValueIdx = i
 		default:
 			// Handle default case
+			// try to convert to *float64
+			var convertedField *data.Field
+			for j := 0; j < field.Len(); j++ {
+				ff, err := field.FloatAt(j)
```

Comment:
I think we want version of this that will work with nulls, so it makes a `*float64` instead of a `float64`, this way, e.g. nullableint64 becomes a nullable float 64

Reply:
fair enough; given that:
> If the Field type is numeric and the value at idx is nil, NaN is returned
I could probably check if the returned value is `math.NaN()` and handled this case or is there any better way?

Reply:
I think there should be a `func (f *Field) NullableFloatAt(idx int) (*float64, error)`, which is pretty much a copy paste of `FloatAt`, but will return a `nil` `*float64` in those cases instead of NaN.